source/java/quickstart/index.markdown

```diff
@@ -110,18 +112,24 @@ The `Client` object is what allows you to communicate with Stormpath.
 
     import com.stormpath.sdk.client.Clients;
     import com.stormpath.sdk.client.Client;
+    import com.stormpath.sdk.client.ClientBuilder;
     import com.stormpath.sdk.api.ApiKeys;
     import com.stormpath.sdk.api.ApiKey;
 
     String path = System.getProperty("user.home") + "/.stormpath/apiKey.properties";
```

Comment:
@marioan @lhazlewood I applied your recommendation. Pull request is again ready for review.